package.json

```diff
@@ -35,7 +36,7 @@
     "glob": "^7.1.1",
     "jest": "^22.0.4",
     "jest-serializer-html": "^4.0.1",
-    "js-yaml": "^3.3.1",
+    "js-yaml": "^3.10.0",
```

Comment:
I didn't realize we were already using `js-yaml`.  I ran `npm install js-yaml --save-dev` and it must've upgraded it.